packages/util/test/logger/cloud-sdk-logger.spec.ts

```diff
@@ -16,6 +18,7 @@ describe('Cloud SDK Logger', () => {
 
   afterEach(() => {
     logger.close();
+    setLogLevel('', messageContext);
```

Comment:
I think that is a fine approach. This could also be a `unsetLogLevel` but most likely this won't be needed.